cli_tools/common/utils/param/populator_test.go

```diff
@@ -267,6 +268,7 @@ func TestPopulator_DeleteResources_WhenScratchBucketInAnotherProject(t *testing.
 		deleteResult         error
 		deleteExpected       bool
 		expectedError        string
+		expectedAnonError    string
```

Comment:
What is "anon"?

Reply:
Anon as in anonymous or anonymized. Updated to expectedAnonymizedError

Reply:
I still see `anon` in tests and in param_utils.go

Reply:
ahh, good find. Updated all.